c/frame/py_frame.cc

```diff
@@ -100,6 +100,51 @@ oobj Frame::copy(const PKArgs&) {
 
 
 
+//------------------------------------------------------------------------------
+// export_names()
+//------------------------------------------------------------------------------
+
+static PKArgs args_export_names(
+  0, 0, 0, false, false,
+  {}, "export_names",
+
+R"(export_names(self)
+--
+
+Create local variables for each column of this frame.
+
+This method inserts into `globals()` as many new variables as there are
+columns in this frame. Each variable is an f-expression referring to
+its column. For example, if the frame has columns A, B, and C, then
+three new variables will be created A = f.A, B = f.B and C = f.C.
+After calling this method you will be able to write column expressions
+using the column names directly, without using the f symbol:
+
+    DT[A + B > C, :]
```

Comment:
What if `A` is a valid column name, but not valid as a Python variable name? Something like `1abc`.

Reply:
You'll have a variable `1abc` in your global scope, though good luck trying to actually use that variable :)

Reply:
Yes, I noticed that. May be worth to mention it in the docs...

Reply:
Nah, those variables are perfectly harmless. 
What could be problematic is when you accidentally overwrite variable `print`, or `str`, or `open`, etc. 
However, this function expects you to be somewhat aware of what columns you have in your frame, and use it only when it's safe to do so.

Reply:
What I mean is writing something like that:

```
After calling this method you will be able to write column expressions
using the column names directly (of course, if the column names are also valid python identifiers):
```